core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -115,6 +120,17 @@ export class DotEditContentFormComponent implements OnInit {
         () => this.$store.contentType()?.fields?.filter((field) => !isFilteredType(field)) ?? []
     );
 
+    /**
+     * Computed property that determines if the preview link should be shown.
+     *
+     * @memberof DotEditContentFormComponent
+     */
+    $showPreviewLink = computed(() => {
+        const contentlet = this.$store.contentlet();
+
+        return contentlet?.baseType === 'CONTENT' && !!contentlet.URL_MAP_FOR_CONTENT;
```

Comment:
is possible to have contentlets different from content with urlMaps?

Reply:
No.